docker-datadog-codesee-bridge-example/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM node:20-bullseye
+
+# Datadog required environment variables
+ENV DD_API_KEY ""
```

Comment:
```suggestion
ENV DD_API_KEY <Your Datadog API Key>
```

Reply:
so we don't want to recommend customer to hard-code their key here, as it will show up in the final image. I left it as empty string so they can override it when they run (ex: `docker run -e DD_API_KEY=123`). 

Maybe to make it clear: 
```
# leave this key as empty but pass in the actual value via docker run. Ex: docker run -e DD_API_KEY=123
ENV DD_API_KEY ""
```

Reply:
Sure, that works